recipe/bld.bat

```diff
@@ -1,42 +1,67 @@
+@echo on
 
-echo "CXXFLAGS: %CXXFLAGS%"
-set "CXXFLAGS=%CXXFLAGS% /FI%RECIPE_DIR%\myiso646.h"
-echo "CXXFLAGS: %CXXFLAGS%"
+:: overwrite Clang with GNU CLI activation
+:: https://github.com/conda-forge/clang-win-activation-feedstock/blob/724a3900c88ef1748a9c1d90d0c2927db99c3ef5/recipe/activate-clang_win-64.bat#L3-L8
+set LDFLAGS=
+set CFLAGS=
+set CXXFLAGS=
```

Comment:
Instead of doing this, you should remove `{{ compiler('clang') }}` and replace it with `clang` and `lld`

Reply:
Thanks for the hint!